File: v-lang/lang.java/vtils/src/main/java/org/example/json/FastjsonUtils.java
package org.example.json;

import com.alibaba.fastjson2.JSON;
import com.alibaba.fastjson2.TypeReference;
import lombok.experimental.UtilityClass;

@UtilityClass
public class FastjsonUtils {

    public <T> T json2Object(String json, Class<T> clazz) {
        return JSON.parseObject(json, clazz);
    }

    public <T> T json2Object(String json, TypeReference<T> typeRef) {
        return JSON.parseObject(json, typeRef);
    }

    public String object2Json(Object obj) {
        return JSON.toJSONString(obj);
    }
}

File: v-lang/l-kotlin/v-utils/u-json/src/main/kotlin/utils/FastjsonUtils.kt
package org.example.utils

import com.alibaba.fastjson2.JSON
import com.alibaba.fastjson2.TypeReference

object FastjsonUtils {

    fun <T> json2Object(json: String, clazz: Class<T>): T = JSON.parseObject(json, clazz)

    fun <T> json2Object(json: String, typeRef: TypeReference<T>): T = JSON.parseObject(json, typeRef)

    fun object2Json(obj: Any): String = JSON.toJSONString(obj)
}